static/js/containers/App_test.js

```diff
@@ -179,4 +182,43 @@ describe('App', () => {
       });
     });
   });
+
+  describe('navbar', () => {
+    for (const [title, url] of [
+      ['Dashboard', '/dashboard'],
+      ['View Profile', `/learner/${SETTINGS.user.username}`],
```

Comment:
Could it ever be the case that `SETTINGS.user` is null? That is, an anonymous user is viewing this page?